src/configure-robot.ts

```diff
@@ -60,6 +60,9 @@ async function getGitHubCIDRs(logger: Logger): Promise<string[]> {
 }
 
 export default async (app: Application): Promise<Application> => {
+
+	app.router.use(elapsedTimeMetrics);
```

Comment:
I moved it to the top level to make sure it will be executed first and capture any error if it happens in the middleware

Reply:
Would it make sense to move it to frontend app (src/frontend/index.ts)? It doesn't work for webhooks anyway , however this is quite confusing to have these on the top level (in my PR I'm relocating all these middlewares to frontend app)...

Reply:
@bgvozdev you'd also want to capture github specific routes since this file calls both github and frontend routes.

Reply:
Yes this, however I don't think webhooks still get there. We need to find the way to extend it to probot (or solve it after we get rid of problt). Hence we'll use the same metric I'd like to leave it as is. 

Also it is important to capture here so there will be as little middleware before `elapsedTimeMetrics` as possible so we'll capture all potential 500s